searx/results.py

```diff
@@ -184,6 +183,7 @@ class ResultContainer:
 
     def __init__(self):
         super().__init__()
+        metrics_initialize()
```

Comment:
Why is the init needed here .. as far I can say metrics should be initialized by the `initialize_metrics` call here in the `searx.search` package

https://github.com/searxng/searxng/blob/cf7627557ab29e31efbf4a0719d1083592e22242/searx/search/__init__.py#L36

Reply:
> get_ordered_results() -> close() -> counter_add() -> counter_storage not defined (or similiar) would somehow fail.

Sure? .. I can't reproduce it .. I commented out `metrics_initialize` and made some tests `make test run ..` without an issue.

Reply:
Something is flaky.
Here it's reproducible and this is one example traceback.

```ERROR: test_result_merge_by_title (tests.unit.test_results.ResultContainerTestCase.test_result_merge_by_title)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/workspaces/searxng/tests/unit/test_results.py", line 72, in test_result_merge_by_title
    self.container.extend('stract', [fake_result(engine='stract', title='short title')])
  File "/workspaces/searxng/searx/results.py", line 243, in extend
    histogram_observe(standard_result_count, 'engine', engine_name, 'result', 'count')
  File "/workspaces/searxng/searx/metrics/__init__.py", line 49, in histogram_observe
    histogram_storage.get(*args).observe(duration)
    ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
```